lib/AST/GenericSignature.cpp

```diff
@@ -1029,6 +1029,14 @@ unsigned GenericParamKey::findIndexIn(
   return genericParams.size();
 }
 
+SubstitutionMap GenericSignature::getIdentitySubstitutionMap() const {
+  return SubstitutionMap::get(const_cast<GenericSignature*>(this),
+                              [](SubstitutableType *t) -> Type {
+                                return Type(cast<GenericTypeParamType>(t));
```

Comment:
Do you mind doing a quick search to see if this utility function could be used anywhere else? I seem to recall writing similar things